### ISSUE TITLE
Dialog actions issue

### ISSUE THREAD

**Issue by colinkong0406** [jacklam718/react-native-popup-dialog#131]
Hi.
Thank you for your update and I've just updated my code as your guide.
But now, I've a problem with actions.
This is the code that I've changed.

````
        <Dialog
                visible={ this.state.dialog_show }
                dialogAnimation={ new ScaleAnimation(0) }
                width={ 0.8 } height={ 0.25 }
                overlayPointerEvents='none'
                dialogTitle={<DialogTitle title={ this.state.dialog_title } />}
                actions={[
                    <DialogButton
                        key="button-1"
                        textStyle={{ fontSize: 13, color: '#000' }}
                        text="CANCEL"
                        onPress={() => {
                            this.setState({ dialog_show: false });
                        }}
                    />,
                    <DialogButton
                        key="button-2"
                        textStyle={{ fontSize: 13, color: '#000' }}
                        text="OK"
                        onPress={() => {
                            this.setState({ dialog_show: false });
                        }}
                    />
                ]}>
````

Also this is the warning from app that is showed up when I show the dialog.
![image](https://user-images.githubusercontent.com/29259395/47619006-257f7280-db0c-11e8-8e9a-8b8e683db6e6.png)

So please help me with this.
This is so urgent problem.
Great regards.

**Comment by jacklam718** [comment]
From the warning, it saying you passing array of object to. But, it expected array of `DialogButton`
But your code seems fine. Maybe try rebuild/rebundle?

**Comment by colinkong0406** [comment]
Thanks for your reply.
But there's still warning.

And I've just tried to run the example from the library.
But it says same.

I think if the example is correct, there's problem in updated version.
I may be wrong...

Anyway, do you have any other suggestion?

**Comment by jacklam718** [comment]
Hi @colinkong0406 Sorry about that. I've fixed it in `v0.16.1`

**Comment by colinkong0406** [comment]
👍 

Really appreciate that!!!
It works!!!

Thank you very much.
